### DISCUSSION TITLE
Contributing DAG examples for PSRPOperator

### DISCUSSION THREAD

**Discussion by arjunanan6** [apache/airflow#32892] [General] [answered]
I've been using the PsrpOperator quite frequently, and want to contribute a few examples. What is the right way of contributing it? I saw some of the current provider examples like the example_dag dir in airflow.providers.microsoft.azure. Is it exactly like that, or is there a template to be followed?

**Comment by potiuk** [comment, marked as answer]
Just take one of those examples that are closest to what you want to do.
